### ISSUE TITLE
Firefox 23 on iOS not detected

### ISSUE THREAD

**Issue by ben4d85** [wilddeer/Sniffer#11]
When accessing your demo page using Mobile Firefox version 23 on iOS (iPhone 10), I get the following result:

```
Sniff.browser.fullName =""
Sniff.browser.name = ""
Sniff.browser.version = ""
Sniff.browser.majorVersion = null
Sniff.browser.minorVersion = null
```

Please can you look into addressing this?
